src/components/Vaults/VaultHistoryRow.tsx

```diff
@@ -24,7 +25,7 @@ export const VaultHistoryRow: React.FC<Props> = ({ vault, config }) => {
     <tr>
       <td className="pl-5 pr-2 py-4 whitespace-nowrap">{formattedTxType}</td>
       <td className="px-2 py-4">
-        {`${ethers.utils.formatEther(vault.amount)}`}
+        {`${formatToFourDecimals(ethers.utils.formatEther(vault.amount))}`}
```

Comment:
and then use `utils.formatting.formatToFourDecimals(args...)` here

Reply:
It doesn’t like the big number

Reply:
```js
utils.formatting.formatToFourDecimals(
    ethers.utils.formatEther(vault.amount)
)
```